fixtures/schema.js

```diff
@@ -73,33 +75,33 @@ var Channel = schema({
   '*': Function
 });
 
-var ChannelKey = String;
+var ChannelKey = NotEmptyString;
 
 var Mode = schema({
-  'name': String,
-  '?shortName': [String, null], // null: use name
+  'name': NotEmptyString,
+  '?shortName': [NotEmptyString, null], // null: use name
```

Comment:
Can we also disallow null to force removing the attribute to use the name, like we did with channel names?